File: backend/components/metiers/threed/utils/samples/samples/routes/samples/samples/examples/sample_route.js
// sample_route.js - Exemple métier avancé route imbriquée
function sampleRoute(user, action) {
  if (!user || !user.role) return false;
  if (user.role === 'admin') return true;
  if (user.role === 'user' && action === 'read') return true;
  return false;
}
module.exports = sampleRoute;

File: backend/components/metiers/threed/utils/samples/samples/routes/samples/samples/examples/sample_route.py
# sample_route.py - Exemple métier avancé route imbriquée

def sample_route(user, action):
    if not user or 'role' not in user:
        return False
    if user['role'] == 'admin':
        return True
    if user['role'] == 'user' and action == 'read':
        return True
    return False
